config/cluster/fsx/config.go

```diff
@@ -18,4 +18,7 @@ func Configure(p *config.Provider) { //nolint:gocyclo
 			Extractor:     common.PathARNExtractor,
 		}
 	})
+	p.AddResourceConfigurator("aws_fsx_ontap_file_system", func(r *config.Resource) {
+		r.LateInitializer.IgnoredFields = []string{"disk_iops_configuration"}
+	})
```

Comment:
Could you please copy this configuration to [config/namespaced/fsx/config.go](https://github.com/crossplane-contrib/provider-upjet-aws/blob/main/config/namespaced/fsx/config.go)?

Reply:
Sure! Done, please check if i did it correctly 😅

Reply:
Please run make generate again :)

Reply:
done :)